src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx

```diff
@@ -4108,6 +4108,47 @@ For tips on configuring logs for the Node.js agent, see [Configure Node.js logs
   </Collapser>
 </CollapserGroup>
 
+## Version tracking [#version-tracking]
+
+
+Setting `NEW_RELIC_METADATA_SERVICE_VERSION` creates a tag, `tag.service.version` on event data. This tag is the version of your deployed code, in many cases a semantic version such as 1.2.3, but not always. Sending this information allows you to facet your telemetry by the version of the software deployed, so you can quickly identify which versions of your software are producing the errors. 
```

Comment:
From the Node.js agent engineers:

We have concerns adding this documentation for an environment variable that currently does not exist in our agent code. The way this is written it appears as if this is something that customers can actually configured and use, while there have been zero code changes at the agent level, nor has this change even been discussed/prioritized with us. If our intent is to "preview" functionality in our docs for upcoming changes, we would advocate that it not be part of the configuration page, and recommend that it is reworded to explicitly state that this is future functionality.

cc @coreyarnold